### PR TITLE
Bugfix debug_info

### PR DESCRIPTION
**Pull request by khalidharun**
@robertzk We need this merged ASAP to unbreak things.

### CODE REVIEW

**Comment by peterhurford** [comment]
:+1: for debugging a debugger

File: R/db.R [hunk 1]
```diff
@@ -407,7 +407,7 @@ get_new_key <- function(dbconn, tbl_name, ids, key) {
   } else {
     shards <- shards[[1]]
   }
-  if (!DBI::dbExistsTable(dbconn, shards[1])) return(ids)
```
Comment (peterhurford) [inline R/db.R:410]: @kirillseva @robertzk , @khalidharun says that right now if the table doesn't exist, it returns `ids` and says that _all_ of them are cached.  Instead, @khalidharun thinks we should return nothing to show that _none_ of them are cached.  Can you verify this?

Comment (khalidharun) [inline R/db.R:410]: I lied.  I mistook this from another refactor I was doing earlier. I'll revert this.

**Comment by robertzk** [comment]
Already in

**Pull request closed without merging**